runtime/CMakeLists.txt

```diff
@@ -133,6 +133,14 @@ else()
 	add_custom_target(j9vm_nlsgen)
 endif()
 
+# Generate jvmti.h
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/include/jvmti.h"
```

Comment:
I think `"${CMAKE_CURRENT_BINARY_DIR}/include/jvmti.h"` needs to be added to the list of dependencies for `j9vm_m4gen` below.